ci-operator/config/openshift/managed-upgrade-operator/openshift-managed-upgrade-operator-master.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.4"
+    namespace: ocp
+    tag: base
```

Comment:
@ravitri we should change this to the `ubi` image and tag `8`

Reply:
Done.. Thanks a lot for this :)